internal/node/node_patches.js

```diff
@@ -95,7 +95,8 @@ const patcher = (fs = fs__default['default'], roots) => {
             args[args.length - 1] = (err, stats) => {
                 if (err)
                     return cb(err);
-                path__default['default'].resolve(args[0]);
+                const pathString = args[0] instanceof Buffer ? args[0].toString() : args[0] instanceof URL ? args[0].toString().replace('file://', '') : args[0];
```

Comment:
If you want to upstream this outside of our org then I think you should pull this into a helper function.